types/util.go

```diff
@@ -0,0 +1,48 @@
+// Copyright 2019 PingCAP, Inc.
```

Comment:
What about put this to `tidb/util/math`. `abs` and `len of int` are so common that it's very likely other places many need them too.

Reply:
I can do it, if someone else also agree with you :) 
Or if it is something for sure you think I should change, please let me know. I can try move the package place.

Reply:
so should I do it or...? I am not sure if it is necessary and whether other places will use these functions.